_episodes/07-Practical Considerations:Reporting-results.md

```diff
@@ -31,30 +34,32 @@ As with any other statistical analysis you can expect to report various metrics
+- **True Negative** - The classifier predicts a negative case i.e. *no* cancer, and that person doesn't actually have cancer.
+- **False Positives** - The classifier predicts a positive case i.e. cancer, but that person doesn't actually have cancer.
+- **True Positives** - The classifier predicts a negative case i.e. *no* cancer, but that person does actually have cancer.
 
-These 4 components can be used to calculate what are the most common metrics in supervised classification - accuracy, sensitivity and specificity. 
+These 4 components can be used to calculate the most common metrics in supervised classification: accuracy, sensitivity and specificity. 
 
-**Accuracy** - The percentage of correct classifications in the dataset
-**Sensitivity** - The percentage of positive cases correctly identified. Sometimes called the *True Positive Rate (TPR)*
-**Specificity** - The percentage of negative cases correctly identified. Sometimes called the *False Positive Rate (FPR)*
+- **Accuracy** - The percentage of correct classifications in the dataset.
+- **Sensitivity** - The percentage of positive cases correctly identified. Sometimes called the *True Positive Rate (TPR)*.
+- **Specificity** - The percentage of negative cases correctly identified. Sometimes called the *True Negative Rate (TNR)*
 
-In some use-cases, such as *information retrieval tasks*, the **precision** of a model can be important where you aren't expecting to evaluate any negative cases. For example if we wrote a machine learning algorithm to scan clinic letters for people with epilepsy, we know that most of the population do not have epilepsy and so "no epilepsy" won't be explicitly recorded. Only *if* someone has epilepsy would it be recorded. So we are interested in assessing out of all the cases the algorithm predicts as having epilepsy, how many actually had epilepsy? but we are also interested in assessing out of all the *possible* epilepsies in the population, what percentage does the algorithm pick up? The latter is measured by precision. Precision is used for *information retrieval* tasks, and sensitivity is used for *classification* tasks. 
+In some use-cases, such as *information retrieval tasks*, the **precision** of a model can be important where you aren't expecting to evaluate any negative cases. For example if we wrote an ML algorithm to scan clinic letters for people with epilepsy, we know that most of the population do not have epilepsy and so "no epilepsy" won't be explicitly recorded. Only *if* someone has epilepsy would it be recorded. So we are interested in assessing out of all the cases the algorithm predicts as having epilepsy, how many actually had epilepsy? but we are also interested in assessing out of all the *possible* epilepsies in the population, what percentage does the algorithm pick up? The latter is measured by precision. Precision is used for *information retrieval* tasks, and sensitivity is used for *classification* tasks. 
```

Comment:
> The latter is measured by precision. 

@arronlacey shouldn't it be `recall`?

> Precision is used for *information retrieval* tasks, and sensitivity is used for *classification* tasks.

- I agree that precision or its other variants, e.g., MAP is used in IR, but we also use precision/recall/F1 for classification tasks.

Reply:
ah sorry I got these mixed up - the former is precision, and the latter is sensitivty = recall (but I want to call it sensitivity as it leads from the discussion on sensitivty and specificity.). I'll need to make it explicit that sensitivity is the same as recall. Thanks for spotting this!

yes true regarding precision in classification (PR curves for imbalanced data etc). I think I will re-write this to emphasize that we tend to find precision a more interesting measure to complement sensitivity/recall in information retrieval tasks.

Reply:
Sounds good, thanks.